### PR TITLE
[enhance-wip](Cloud) Set storage vault id for all load paths

### PR DESCRIPTION
**Pull request by ByteYue**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: be/src/runtime/load_stream.h [hunk 1]
```diff
@@ -80,7 +80,7 @@ class IndexStream {
 public:
     IndexStream(PUniqueId load_id, int64_t id, int64_t txn_id,
                 std::shared_ptr<OlapTableSchemaParam> schema, LoadStreamMgr* load_stream_mgr,
-                RuntimeProfile* profile);
+                RuntimeProfile* profile, std::string storage_vault_id);
```
Comment (gavinchou) [inline be/src/runtime/load_stream.h:83]: const ref?

**Review by gavinchou — APPROVED** [review]

File: be/src/runtime/load_stream.h (hunk 1 above)
Comment (gavinchou) [reply be/src/runtime/load_stream.h:83]: need to change

**Review by gavinchou — CHANGES_REQUESTED** [review]

**Comment by dataroaring** [comment]
run buildall

**Comment by dataroaring** [comment]
run buildall

**Pull request closed without merging**